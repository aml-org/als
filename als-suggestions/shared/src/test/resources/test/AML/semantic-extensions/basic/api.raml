#%RAML 1.0
title: semantic test
version: 1
(*