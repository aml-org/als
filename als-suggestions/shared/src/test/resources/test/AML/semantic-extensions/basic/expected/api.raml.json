[
  {
    "label": "(maintainer)",
    "kind": [],
    "detail": "extensions",
    "documentation": "(maintainer)",
    "deprecated": [],
    "preselect": [],
    "sortText": "21(maintainer)",
    "filterText": "(maintainer):\n  ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 0
        },
        "end": {
          "line": 3,
          "character": 1
        }
      },
      "newText": "(maintainer):\n  "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "(rateLimiting)",
    "kind": [],
    "detail": "extensions",
    "documentation": "(rateLimiting)",
    "deprecated": [],
    "preselect": [],
    "sortText": "21(rateLimiting)",
    "filterText": "(rateLimiting): ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 0
        },
        "end": {
          "line": 3,
          "character": 1
        }
      },
      "newText": "(rateLimiting): "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]